### ISSUE TITLE
[Feature Request]: DUAL GPU

### ISSUE THREAD

**Issue by krigeta** [lllyasviel/stable-diffusion-webui-forge#48]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

can use a Dual GPU setup.

### Proposed workflow

no workflow

**Comment by lllyasviel** [comment]
I added `--gpu-device-id` : https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/74ff4a9ba95ad1d92d48f7bac5f0de8c0c15e398

**Comment by krigeta** [comment]
> I added `--gpu-device-id` : [74ff4a9](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/74ff4a9ba95ad1d92d48f7bac5f0de8c0c15e398)

great, how can I use it so it can able to use two graphics card? `--gpu-device-id:0,1` ?

**Comment by SaturMars** [comment]
> > I added `--gpu-device-id` : [74ff4a9](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/74ff4a9ba95ad1d92d48f7bac5f0de8c0c15e398)
> 
> great, how can I use it so it can able to use two graphics card? `--gpu-device-id:0,1` ?

--gpu-device-id 0 or --gpu-device-id 1

It seems that the two GPUs can't work together at the same time, but when using multidiffusion to generate huge images, is it worth considering distributing different tiles to multiple GPUs?

**Comment by krigeta** [comment]
> > > I added `--gpu-device-id` : [74ff4a9](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/74ff4a9ba95ad1d92d48f7bac5f0de8c0c15e398)
> > 
> > 
> > great, how can I use it so it can able to use two graphics card? `--gpu-device-id:0,1` ?
> 
> --gpu-device-id 0 or --gpu-device-id 1
> 
> It seems that the two GPUs can't work together at the same time, but when using multidiffusion to generate huge images, is it worth considering distributing different tiles to multiple GPUs?

thank you and I don't know about that

**Comment by ShadowPlague21** [comment]
networked distributed inference capabilities when? :-)

**Comment by jswag245** [comment]
@krigeta, hi, just checking to see if the issue has been fixed/solved to any extent. if it is not possible for two GPU's to work together at the same time, and that answer is satisfactory for you, please close the issue.

**Comment by krigeta** [comment]
> this issue should stay open because it is possible, just not with the current implementation. sdcpp has shown that its possible via cublas, and accelerate would make it possible for pytorch, but nobody has done it for a pytorch based ui such as comfyui and webui, only for cpp and c# implementations so far.

I agree, I was about to comment on it, but thanks @yggdrasil75 for this, I hope someday it will be added, till then people must know that as of now it is not possible so that is why this issue is still open.
